docs/quickstart.rst

```diff
@@ -338,6 +338,45 @@ Or you can ignore missing fields entirely by setting ``partial=True``.
     # OR UserSchema(partial=True).load({'age': 42})
     result  # => ({'age': 42}, {})
 
+.. _unknown:
+
+Handling Unknown Fields
++++++++++++++++++++++++
+
+By default, :meth:`load <Schema.load>` will drop any field that has not defined in the schema.
+
+This behavior can be modified with the ``unknown`` option, which accept three values:
+
+- `DROP <marshmallow.utils.DROP>`, which is the default
```

Comment:
These need to be updated.

Reply:
Also, this should explain what the option does, for consistency.

```
- `EXCLUDE <marshmallow.utils.EXCLUDE>`, to exclude unknown fields from the returned data. This is the default behavior.
```

Reply:
Done